api/events/enroll.py

```diff
@@ -55,13 +54,16 @@ class EnrollResponseModel(OPModel):
 @router.post("/enroll", response_model=EnrollResponseModel)
```

Comment:
I don't think so. 204 is not the "Primary" response. What you expect is EnrollResponseModel and 204 is treated more or less like 404 (but error code is not used since it would flood the logs). But response_model argument is redundant here.